sklearn/gaussian_process/kernels.py

```diff
@@ -1348,9 +1357,9 @@ def __repr__(self):
                 self.__class__.__name__,
                 ", ".join(map("{0:.3g}".format, self.length_scale)),
                 self.nu)
-        else:  # isotropic
+        else:
             return "{0}(length_scale={1:.3g}, nu={2:.3g})".format(
-                self.__class__.__name__, self.length_scale, self.nu)
+                self.__class__.__name__, _squeeze(self.length_scale), self.nu)
```

Comment:
I remimplemented `np.squeeze` because `.format` does not play well with a 0-D array.

```python
a = "{0:0.3g}".format(np.array(2))
TypeError: non-empty format string passed to object.__format__
a = "%0.3g" % np.array(2)
a
'2'
```

Probably should be reported to NumPy...